plugins/huaweicloud/src/main/java/org/eclipse/xpanse/plugins/huaweicloud/common/HuaweiCloudClient.java

```diff
@@ -125,6 +134,45 @@ public BssClient getBssClient(ICredential globalCredential) {
                 .build();
     }
 
+    /**
+     * Get HuaweiCloud Bss intl Client.
+     *
+     * @param globalCredential ICredential
+     */
+    public BssintlClient getBssintlClient(ICredential globalCredential) {
+        return BssintlClient.newBuilder()
+                .withHttpConfig(getHttpConfig())
+                .withCredential(globalCredential)
+                .withRegion(BssintlRegion.AP_SOUTHEAST_1)
```

Comment:
@baixinsui is this hardcoding of region necessary? Because prices are region dependent. 

If this is really necessary, then we must add a comment on why we do it.